src/main/java/frc/robot/command/CollisionDetection.java

```diff
@@ -0,0 +1,197 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.command;
+
+import com.ctre.phoenix6.Utils;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Constants.CollisionConstants;
+import frc.robot.Constants.TunerConstants;
+import frc.robot.Constants.CollisionConstants.CollisionType;
+import frc.robot.subsystems.Swerve;
+import edu.wpi.first.math.filter.LinearFilter;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.util.Units;
+import frc.thunder.shuffleboard.LightningShuffleboard;
+
+public class CollisionDetection extends Command {
+
+    private Swerve drivetrain;
+
+    public Rotation2d collisionDirection;
```

Comment:
is this value ever used?

Reply:
no, its just meant to be something that can be used, but i got rid of it since it probably wont be vary accurate